### DISCUSSION TITLE
Library no pictures found / Folder is empty / photoprism originals empty

### DISCUSSION THREAD

**Discussion by addmedia-cottbus** [photoprism/photoprism#3872] [Q&A]
Hi there,

after setting up photoprism 3 days ago i was happy that indexing finally was finished.

Would be cool, but now photoprism lost the connection to the originals folder, apparently it could open the other folders which have the same premissions.
![ffBeyoL6Ow](https://github.com/photoprism/photoprism/assets/103063523/9f9fae3e-ce21-4813-9df4-64ed59095868)

its all on windows, doesnt even work if i grant rights for everybody (in windows).

Log file is like this:

![rxiVIhdCZ6](https://github.com/photoprism/photoprism/assets/103063523/04cac04f-d10c-481e-b005-10c2ea872f76)

Even weirder:
If i click on a label it says "no pictures found"

if i click on a person it shows me one image, but when i want to open it nothing happens
![gWoMGvV1pM](https://github.com/photoprism/photoprism/assets/103063523/c21d3fc2-3afa-427f-abb2-cd8fc9088067)

i went back to post this to here. went back to people, from there back to my exampleperson which now only had 7 instead of the previous 200something.

WTF is going on here?

**Comment by lastzero** [comment]
Using Portainer?

It is important that all folders are located on a persistent volume on your device - ideally specified with an absolute path - to avoid potential data loss after a restart, e.g. if the default application folder managed by Portainer changes or gets lost for a reason beyond our control.
